kfs-cam/src/main/resources/edu/arizona/kfs/module/cam/ojb-cam.xml

```diff
@@ -0,0 +1,57 @@
+<descriptor-repository version="1.0">
```

Comment:
I don't see why this OJB table override is needed when the new edu.arizona BO class extends org.kuali.kfs.module.cam.businessobject.AssetRetirementGlobal and there are no changed or new database fields or relationships included in this modification.